### PR TITLE
Update Chinese (Simplified) translation

### PR DESCRIPTION
**Pull request by gtn1024**
I translated a part of the strings. I will add the missing translation later.

### CODE REVIEW

**Comment by waydabber** [comment]
Amazing, thank you! Let us know when you are ready so we can approve the merge!

File: MonitorControl/UI/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -20,10 +20,10 @@
 "Build" = "Build";
 
 /* Shown in the Display Preferences */
-"Built-in Display" = "Built-in Display";
+"Built-in Display" = "内建屏幕";
```
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Localizable.strings:23]: 内建显示器是不是更好？和 macOS 设置菜单里的名称保持一致。

File: MonitorControl/UI/zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -32,105 +32,105 @@
...
-"This display has an unspecified control status." = "This display has an unspecified control status.";
+"This display has an unspecified control status." = "该显示器具有未指定的控制状态。";
 
 /* Shown in the Display Preferences */
-"This display is reported to support hardware DDC control but the current settings allow for software control only." = "This display is reported to support hardware DDC control but the current settings allow for software control only.";
+"This display is reported to support hardware DDC control but the current settings allow for software control only." = "该显示器支持硬件 DDC 控制，但当前设置仅允许软件控制。";
 
 /* Shown in the Display Preferences */
-"This display is reported to support hardware DDC control. If you encounter issues, you can disable hardware DDC control to force software control." = "This display is reported to support hardware DDC control. If you encounter issues, you can disable hardware DDC control to force software control.";
+"This display is reported to support hardware DDC control. If you encounter issues, you can disable hardware DDC control to force software control." = "该显示器支持硬件 DDC 控制。如果你遇到问题，你可以关闭硬件 DDC 控制强制使用软件控制。";
```
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Localizable.strings:110]: > 如果你遇到问题，你可以关闭硬件 DDC 控制强制使用软件控制。

长句子能否加个逗号？主语也不需要重复出现。

如果遇到问题，你可以关闭硬件 DDC 控制，强制使用软件控制。

File: MonitorControl/UI/zh-Hans.lproj/Localizable.strings [hunk 3]
```diff
@@ -32,105 +32,105 @@
...
 
 /* Shown in menu */
 "Volume" = "音量";
 
 /* Shown in the alert dialog */
-"Yes" = "Yes";
+"Yes" = "是";
 
 /* Shown in the alert dialog */
 "You need to enable MonitorControl in System Preferences > Security and Privacy > Accessibility for the keyboard shortcuts to work" = "您需要在「系统偏好设置」>「安全性与隐私」>「辅助功能」中开启 MonitorControl 以便使快捷键生效";
```
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Localizable.strings:136]: > 开启 MonitorControl 以便使快捷键生效。

这里有语病，简化一下就行了。

开启 MonitorControl 令快捷键生效。

File: MonitorControl/UI/zh-Hans.lproj/Main.strings [hunk 4]
```diff
@@ -1,62 +1,62 @@
...
-"7rn-Lu-fcl.title" = "Use combined slider for all displays";
+"7rn-Lu-fcl.title" = "对所有显示使用组合滑块";
 
 /* Class = "NSButtonCell"; title = "Show slider tick marks"; ObjectID = "7zf-m1-gJO"; */
-"7zf-m1-gJO.title" = "Show slider tick marks";
+"7zf-m1-gJO.title" = "显示滑块标记";
 
 /* Class = "NSTextFieldCell"; title = "Slider knob will snap to 0%, 25%, 50%, 75% and 100% when in proximity making setting these values easier. Disable for finer control."; ObjectID = "8Gx-Ya-zhp"; */
-"8Gx-Ya-zhp.title" = "Slider knob will snap to 0%, 25%, 50%, 75% and 100% when in proximity making setting these values easier. Disable for finer control.";
+"8Gx-Ya-zhp.title" = "当滑块靠近 0%、25%、50%、75%、100% 时，会自动到达。禁用以获得更好的控制。";
```
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Main.strings:47]: > 当滑块靠近 0%、25%、50%、75%、100% 时，会自动到达。禁用以获得更好的控制。

这句并不能让人理解预期行为。

滑块将自动吸附 0%、25%、50%、75%、100% 。禁用此选项进行更精准的控制。

File: MonitorControl/UI/zh-Hans.lproj/Localizable.strings [hunk 5]
```diff
@@ -32,105 +32,105 @@
...
-"No Control" = "No Control";
+"No Control" = "无控制";
 
 /* Shown in the Display Preferences */
-"Other Display" = "Other Display";
+"Other Display" = "其他显示";
 
 /* Shown in the alert dialog */
-"Preferences for an incompatible previous app version detected. Default preferences are reloaded." = "Preferences for an incompatible previous app version detected. Default preferences are reloaded.";
+"Preferences for an incompatible previous app version detected. Default preferences are reloaded." = "检测到先前不兼容的应用程序版本的偏好设置，默认偏好设置已被重新加载。";
```
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Localizable.strings:71]: 有点绕口...这样是不是更好。

检测到不兼容的旧版应用程序偏好设置，已重新加载默认偏好设置。

File: MonitorControl/UI/zh-Hans.lproj/Main.strings [hunk 6]
```diff
@@ -1,62 +1,62 @@
 /* Class = "NSButtonCell"; title = "Sync brightness changes from Built-in and Apple displays"; ObjectID = "0ca-DG-AgB"; */
-"0ca-DG-AgB.title" = "Sync brightness changes from Built-in and Apple displays";
+"0ca-DG-AgB.title" = "从内建的和苹果屏幕同步亮度变化";
```
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Main.strings:2]: 这样是不是更好？

同步内建显示器与苹果显示器的亮度变化。

File: MonitorControl/UI/zh-Hans.lproj/Localizable.strings [hunk 7]
```diff
@@ -32,105 +32,105 @@
...
-"Software (Gamma, Forced)" = "Software (Gamma, Forced)";
+"Software (Gamma, Forced)" = "软件 (Gamma, Forced)";
 
 /* Shown in the Display Preferences */
-"Software (Shade)" = "Software (Shade)";
+"Software (Shade)" = "软件 (Shade)";
 
 /* Shown in the Display Preferences */
-"This display allows for software brightness control via gammatable manipulation as it does not support hardware control. Reasons for this might be using the HDMI port of a Mac mini (which blocks hardware DDC control) or having a blacklisted display." = "This display allows for software brightness control via gammatable manipulation as it does not support hardware control. Reasons for this might be using the HDMI port of a Mac mini (which blocks hardware DDC control) or having a blacklisted display.";
+"This display allows for software brightness control via gammatable manipulation as it does not support hardware control. Reasons for this might be using the HDMI port of a Mac mini (which blocks hardware DDC control) or having a blacklisted display." = "因为它不支持硬件控制，该显示器允许通过伽马表操作进行软件亮度控制。原因可能是使用 Mac mini 的 HDMI 端口（阻止硬件 DDC 控制）或具有列入黑名单的显示器。";
```
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Localizable.strings:101]: 这样是不是更好？

此显示器只允许操作伽马表进行软件亮度控制，不支持硬件控制。可能因为连接了 Mac Mini 的 HDMI 端口（屏蔽硬件 DCC 控制），或此显示器已被列入黑名单。

File: MonitorControl/UI/zh-Hans.lproj/Localizable.strings (hunk 3 above)
Comment (moriwang) [inline MonitorControl/UI/zh-Hans.lproj/Localizable.strings:136]: 开启 MonitorControl 以便使快捷键生效。
这里有语病，简化一下就行了。

开启 MonitorControl 令快捷键生效。

**Comment by moriwang** [comment]
@gtn1024  Wait...Are you using Google Translate to finish this work? Seriously? Please stop this KPI grabbing behavior.

**Comment by waydabber** [comment]
> @gtn1024 Wait...Are you using Google Translate to finish this work? Seriously? Please stop this KPI grabbing behavior.

Well, while using Google Translate certainly makes things faster, I'd still advise to at least review and fix Google Translate results instead of simply running the text through it. We (the devs) have no way of assessing the quality of the translations, so I am glad that you guys do proofread each other's work and give healthy advice to each other on this. :)

**Comment by gtn1024** [comment]
@moriwang Alright, I must acknowledge that I did use use Google Translate for some sentence. I will not do it again.

@waydabber Thanks.

**Comment by womeimingzi11** [comment]
Hi@waydabber, I totally agree with @moriwang. It looks @gtn1024 finish all the work by translation tool. I do not think it is a good practice and I highly recommend to totally refuse this PR.

Because, for further improvements, we have to fix all the mistakes which is caused by the non-sense approach.

Please respect all the contributions from others, @gtn1024 . The first PR of Chinese localization is really not so important.

**Comment by waydabber** [comment]
Uhh-ohh, I can't really judge on this case I am afraid, but I understand that an action must be taken, which will be the following:

- Because of the controversy I'll close this PR.
- I'd like to kindly ask any three of you to start a new PR when any of you (or somebody else) completed a full, valid translation (preferably not via an automated tool like Google Translate by itself).
- Because of the questions, if this person will be @gtn1024 (who I am sure has the best of intentions even though it seems the implementation suffered a bit), I'll kindly ask a second opinion from @womeimingzi11 or @moriwang just to make sure that the quality is the expected one this time.
- Please let the others know if somebody starts working on an alternate translation in the [translation thread](https://github.com/MonitorControl/MonitorControl/discussions/637) so we can avoid conflicts and working in vain. If there are multiple translations for the same language and all of them seem legit, I'll choose the one with the lower PR number.

I hope this is is a sensible and fair approach. I am sorry if I misjudged the situation, please let me know if anybody feels bad about this!

Thank you all very much for your contribution!

**Comment by gtn1024** [comment]
I feel so sorry for everybody. Thanks for your work! @moriwang @womeimingzi11 @waydabber

**Comment by womeimingzi11** [comment]
> Uhh-ohh, I can't really judge on this case I am afraid, but I understand that an action must be taken, which will be the following:
> 
> * Because of the controversy I'll close this PR.
> * I'd like to kindly ask any three of you to start a new PR when any of you (or somebody else) completed a full, valid translation (preferably not via an automated tool like Google Translate by itself).
> * Because of the questions, if this person will be @gtn1024 (who I am sure has the best of intentions even though it seems the implementation suffered a bit), I'll kindly ask a second opinion from @womeimingzi11 or @moriwang just to make sure that the quality is the expected one this time.
> * Please let the others know if somebody starts working on an alternate translation in the [translation thread](https://github.com/MonitorControl/MonitorControl/discussions/637) so we can avoid conflicts and working in vain. If there are multiple translations for the same language and all of them seem legit, I'll choose the one with the lower PR number.
> 
> I hope this is is a sensible and fair approach. I am sorry if I misjudged the situation, please let me know if anybody feels bad about this!
> 
> Thank you all very much for your contribution!

Good idea @waydabber, I totally agree with this plan. In fact,I didn't translate by myself, so there was no conflict. I just checked the PR list to find what's new here. 

Can't wait for @gtn1024 's new contribution! Good luck!

BTW, I realized there may need some peer-review-like approaches in translation workflow. For some more detailed suggestions I'd like to post in discussion later.

**Comment by moriwang** [comment]
Seems like #657 took some of my suggestion, maybe someone can continues work on the that.

**Pull request closed without merging**